### ISSUE TITLE
Add support for multiple paths in --source flag

### ISSUE THREAD

**Issue by phun-ky** [mattmcmanus/dox-foundation#21]
From what I see in the docs and code, you can't pass multiple paths to dox-foundation:

I would like this to work:

```
$ dox-foundation --source "<folder1>, <folder2>, <filepath>, <folder3>" --target <path to target>
```

Tried to manipulate the code myself to support it, but was more time consuming that I thought. 

The only solution I see now to solve this without rewriting dox-foundation to support the outcome I require is to never have `*.js`-files in subdirectories..

**Comment by mattmcmanus** [comment]
This is a good idea. It shouldn't be too hard to implement.

**Comment by phun-ky** [comment]
Just to clarify:

```
$ dox-foundation --source <folder with subfolders> --target <path to target>
```

This works.

**Comment by mattmcmanus** [comment]
Yeah. That's always worked (at least on OSX). Adding in walkdir fixed it for windows. You could potentially tell it to look at the root of your project and then specify `--ignore` to have it not parse certain directories for now.

**Comment by phun-ky** [comment]
Stay tuned, the PR: https://github.com/punkave/dox-foundation/pull/24 will contain multidirectories next week ;)

**Comment by phun-ky** [comment]
Updated PR #24 , please test and merge in if you see fit to do so.

**Comment by phun-ky** [comment]
This can be closed now i think, with PR #24 is merged in

**Comment by mattmcmanus** [comment]
Yes! Thanks for the reminder.
